File: kattis/chanukah/chanukah.py
from sys import stdin

# Number of datasets
n = int(stdin.readline().rstrip())

# Process each dataset
for i in range(n):
    # The number of days to find the total candles for up to (and including)
    val = int(stdin.readline().rstrip().split(" ")[1])

    # Quick summation from 1 to i, for the building amount each day, + the day i,
    # to represent the shammas, one of which is lit every day
    candles = int(((val * (val + 1)) / 2) + val)
    print(str(i + 1) + " " + str(candles))

File: kattis/chanukah/chanukah.cpp
#include <iostream>

using namespace std;

int main() {
    int P;
    cin >> P;

    while (P--) {
        int K, N, candles;
        cin >> K >> N;

        candles = (N * (N + 1)) / 2 + N;
        cout << K << " " << candles << endl;
    }
}